src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -96,7 +102,7 @@ export const Section: React.FC<SectionProps> = ({ venue }) => {
           <div className={centralScreenClasses}>
             <IFrame containerClassName="Section__iframe" src={iframeUrl} />
             <div className="Section__reactions">
-              {isUserSeated ? (
+              {shouldShowReactions ? (
```

Comment:
please, check this part.
I've turned off the reactions and took a seat, but the message under the video is still "Welcome! Click on an empty seat to claim it!"

<img width="1390" alt="Screenshot 2021-09-15 at 16 56 38" src="https://user-images.githubusercontent.com/63313732/133447186-3ecf1449-0349-42c7-9f40-7b2f5f1f2ef8.png">

Better behavior would be if there will be just "Welcome!" or no text at all. (you can check what looks better)